sdk/cosmosdb/Azure.ResourceManager.CosmosDB/tests/ScenarioTests/PrivateEndpointConnectionTests.cs

```diff
@@ -59,7 +59,6 @@ public async Task TearDown()
         }
 
         [Test]
```

Comment:
Could we leave `RecordedTest` attribute and remove `Test` attribute here instead?

Reply:
fixed